src/ucp/wireup/wireup_cm.c

```diff
@@ -275,9 +275,53 @@ ucp_cm_ep_sa_data_pack(ucp_ep_h ep, ucp_wireup_sockaddr_data_base_t *sa_data,
     }
 }
 
+static ucs_status_t
+ucp_cm_ep_priv_space_is_enough(ucp_ep_h ep, const ucp_tl_bitmap_t *tl_bitmap,
+                               ucs_log_level_t log_level,
+                               unsigned ep_init_flags,
+                               ucp_object_version_t sa_data_version)
+{
+    ucp_worker_h worker   = ep->worker;
+    ucp_context_h context = worker->context;
+    unsigned pack_flags   = ucp_cm_address_pack_flags(worker);
+    size_t ucp_addr_size, sa_data_length;
+    ucp_rsc_index_t cm_idx;
+    ucs_status_t status;
+
+    ucs_assert((int)ucp_ep_config(ep)->key.err_mode <= UINT8_MAX);
```

Comment:
what does this assertion check?

Reply:
Removed.
After checking the logic, this is not needed.